prerelease_website/templates/generate_command.html

```diff
@@ -145,7 +145,7 @@ <h3 class="panel-title">Setup your system:</h3>
             </div>
             <div class="panel-body">
               Before running the generated command, you need to setup your system.<br/>
-              You'll find instructions for this at this website (Ubuntu 16.04 recommended, Ubuntu 14.04 also supported):
+              You'll find instructions for this at this website (Ubuntu 18.04 recommended, Ubuntu 16.04 and Ubuntu 14.04 also supported):
```

Comment:
Well 14.04 is no longer supported.

```suggestion
              You'll find instructions for this at this website (Ubuntu 18.04 recommended, Ubuntu 16.04 is also supported):
```